src/xmipp/libraries/reconstruction/image_odd_even.cpp

```diff
@@ -142,7 +133,13 @@ void ProgOddEven::run()
 			movieEven.setValue(MDL_IMAGE, fnFrame, objId_even);
 			if (sumFrames)
 			{
-				imgEven() += frame();
+				for (size_t i =0; i<Ydim;++i)
```

Comment:
so the performance is improved by this? If yes, we should check why

Reply:
No that was the results of other trials I carried out. The change in the performance comes because I removed two unnecessary reading of images.

Reply:
I see. Wouldn't it be better then to skip this change?

Reply:
I agree, however, I do not see any other reason responsible of failing. Now it works, it is an innocent loop :)